src/aura/STG_CMP_Addr/STG_CMP_Addr.cmp

```diff
@@ -89,39 +87,42 @@
                 </div>
             </div>
         </div>
+        <aura:if isTrue="{!v.hierarchySettings.Enable_New_Preferred_Phone_Sync__c}">
+            <div class="slds-col slds-size--1-of-1 slds-m-bottom--medium">
+                <ui:outputText value="{!$Label.c.stgHelpContactPreferredPhoneSync}" class="slds-text-body--small" />
```

Comment:
I think we should still display the help text of the checkbox when it's not enabled.